src/diff/children.js

```diff
@@ -280,12 +275,11 @@ export function toChildArray(children, out) {
 
 /**
  * @param {import('../internal').PreactElement} parentDom
- * @param {number} oldChildrenLength
  * @param {import('../internal').PreactElement} newDom
  * @param {import('../internal').PreactElement} startDom
  * @returns {import('../internal').PreactElement}
  */
-function placeChild(parentDom, oldChildrenLength, newDom, startDom) {
```

Comment:
So happy to see the `oldChildrenLength` parameter go away haha. Other bothered me lol